android/app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,7 @@
 <resources>
     <string name="app_name" translatable="false">Open Event</string>
 
+    <string name="url">2017.fossasia.org</string>
```

Comment:
Shouldn't we take this from the generator in some way, so that this matches with the data coming to the generator? 
A corresponding change to the generator is needed too.